modules/tinymce/src/themes/silver/main/ts/alien/NotificationManagerImpl.ts

```diff
@@ -82,7 +82,10 @@ export default (
     const notification = GuiFactory.build(
       Notification.sketch({
         text: settings.text,
-        level: Arr.contains([ 'success', 'error', 'warning', 'warn', 'info' ], settings.type) ? settings.type : undefined,
+        level: Optional.from(settings.type).fold(
+          () => 'info',
+          (level) => Arr.contains([ 'success', 'error', 'warning', 'warn', 'info' ], level) ? level : undefined
+        ),
```

Comment:
I'd like to extend this requirement to "fall back with an incorrect type" as well. Use `.filter()` to check whether the level is contained in the array.

Whether that's accepted or not, I suggest not using `fold` - in this case `.getOr('info')` will work quite well.

Reply:
I think it's a good idea to handle incorrect notification type as well. I've handled this case as well and added automated test